endpoints/get_address_transactions.py

```diff
@@ -88,7 +102,9 @@ async def get_full_transactions_for_address(
             ge=0,
             default=0),
         fields: str = "",
-    ):
+        resolve_previous_outpoints: PreviousOutpointLookupMode =
+        Query(default="no",
```

Comment:
Instead of using `"no"` directly here, which is the value of the enum, use the Enum property like this `PreviousOutpointLookupMode.no`.

This asserts that the value you're using as a default exists in the enum

Reply:
Yep, fixed this!